STAT/TStatToolkit.cxx

```diff
@@ -1143,6 +1143,7 @@ TGraph * TStatToolkit::MakeGraphSparse(TTree * tree, const char * expr, const ch
     ::Error("TStatToolkit::MakeGraphSparse","Empty or Not valid expression (%s) or cut (%s)", expr, cut);
     return 0;
   }
+  else ::Info("TStatToolkit::MakeGraphSparse","Valid expression (%s) or cut (%s)", expr, cut);
```

Comment:
Should be Debug

Reply:
I deleted this line since its information was not useful - expression and cut were not the problem